app/services/uri_service.rb

```diff
@@ -14,8 +14,8 @@ def self.add_params(original_uri, params_to_add)
     return if original_uri.blank?
 
     URI(original_uri).tap do |uri|
-      query = params(uri)
-      uri.query = query.merge(params_to_add).to_query
+      query = params(uri).merge(params_to_add || {})
```

Comment:
This can be reverted if preferred. I added this in an effort to make the code more flexible. The original version will fail fast.

Should a mistake be made, the original version of the code might result in "undefined method `to_hash' for nil".

Reply:
I personally don't think we need to implement support for a value we're not expecting to pass.

If we update this to support `nil`, can we also update the method signature documentation on line 11 to indicate that the parameter can be something other than `Hash`?